src/hooks/UrqlUseSubscription.re

```diff
@@ -17,20 +17,21 @@ type useSubscriptionArgs = {
   context: UrqlClient.ClientTypes.partialOperationContext,
 };
 
+type executeSubscriptionJs;
+
 [@bs.module "urql"]
 external useSubscriptionJs:
   (useSubscriptionArgs, option((option('acc), Js.Json.t) => 'acc)) =>
-  array(UrqlTypes.jsResponse('ret, 'extensions)) =
+  (UrqlTypes.jsResponse('ret, 'extensions), executeSubscriptionJs) =
```

Comment:
The Urql documentation seemed to indicate this was a tuple, and I think this is okay as arrays are the underlying representation of tuples.

Reply:
I believe it's ok too, you could even only have `(UrqlTypes.jsResponse('ret, 'extensions), 'executeSubscriptionJs)` instead of declaring a concrete type for `executeSubscriptionJs` since we aren't using it anyway

Reply:
I think if we were to use a type variable here, and someone were to actually use it, the compiler would just infer it to be whatever it was at the call site and make the program unsafe. While unlikely, my reasoning was that it should be clear that it's not typed yet. Someone could always cast to whatever they need with `Obj.magic` or `%identity` if they needed it in a hurry.

Reply:
Sounds perfectly fine to me then!